<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsModule</class>
 <widget class="QWidget" name="SettingsModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interfaceTab">
      <attribute name="title">
       <string>界面设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="displayGroup">
         <property name="title">
          <string>显示设置</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelLanguage">
            <property name="text">
             <string>语言:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="languageCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDarkMode">
            <property name="text">
             <string>深色模式:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="darkModeCheck"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelTooltips">
            <property name="text">
             <string>显示工具提示:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="showTooltipsCheck"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pathsTab">
      <attribute name="title">
       <string>路径设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="pathGroup">
         <property name="title">
          <string>文件路径</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDefaultPath">
            <property name="text">
             <string>默认保存路径:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="defaultPathEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseDefaultPathButton">
              <property name="text">
               <string>浏览...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelTemplatePath">
            <property name="text">
             <string>模板路径:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="templatePathEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseTemplatePathButton">
              <property name="text">
               <string>浏览...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelDatabasePath">
            <property name="text">
             <string>数据库路径:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="databasePathEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseDatabasePathButton">
              <property name="text">
               <string>浏览...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="settingPathEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="settingPathButton">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>设置保存路径:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cameraTab">
      <attribute name="title">
       <string>相机设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="cameraDefaultsGroup">
         <property name="title">
          <string>默认相机参数</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="labelExposure">
            <property name="text">
             <string>曝光时间(μs):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="exposureSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGain">
            <property name="text">
             <string>增益:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="gainSpin">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>48</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelWhiteBalance">
            <property name="text">
             <string>自动白平衡:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="whiteBalanceCheck"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>相机类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>单目相机</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>双目相机</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>自动采集间隔(ms):：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="calibrationTab">
      <attribute name="title">
       <string>标定设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="calibrationDefaultsGroup">
         <property name="title">
          <string>默认标定参数</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="labelBoardWidth">
            <property name="text">
             <string>棋盘格宽度(内角点):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="boardWidthSpin">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelBoardHeight">
            <property name="text">
             <string>棋盘格高度(内角点):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="boardHeightSpin">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSquareSize">
            <property name="text">
             <string>棋盘格大小(mm):</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="squareSizeSpin">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="restoreDefaultsButton">
       <property name="text">
        <string>恢复默认设置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveSettingsButton">
       <property name="text">
        <string>保存设置</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseDefaultPathButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsModule</receiver>
   <slot>onBrowseDefaultPathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseTemplatePathButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsModule</receiver>
   <slot>onBrowseTemplatePathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseDatabasePathButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsModule</receiver>
   <slot>onBrowseDatabasePathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restoreDefaultsButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsModule</receiver>
   <slot>onRestoreDefaultsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsModule</receiver>
   <slot>onCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveSettingsButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsModule</receiver>
   <slot>onSaveSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBrowseDefaultPathClicked()</slot>
  <slot>onBrowseTemplatePathClicked()</slot>
  <slot>onBrowseDatabasePathClicked()</slot>
  <slot>onRestoreDefaultsClicked()</slot>
  <slot>onSaveSettingsClicked()</slot>
  <slot>onCancelClicked()</slot>
 </slots>
</ui>
